bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BatchExecutor.java

```diff
@@ -97,15 +93,8 @@ public RpcResultFutureCallback(
     @Override
     public final void onSuccess(Object message) {
       Result result = Result.EMPTY_RESULT;
-
       try {
-        if (message instanceof FlatRow) {
-          result = Adapters.FLAT_ROW_ADAPTER.adaptResponse((FlatRow) message);
-        } else if (message instanceof com.google.cloud.bigtable.data.v2.models.Row) {
-          result =
-              Adapters.ROW_ADAPTER.adaptResponse(
-                  (com.google.cloud.bigtable.data.v2.models.Row) message);
-        } else if (message instanceof Result) {
+        if (message instanceof Result) {
```

Comment:
What else can this be?

Reply:
It could be Void as well from BufferedMutatorHelper#mutate:

https://github.com/googleapis/java-bigtable-hbase/blob/792fe728b6feaa67ee3229f2ada22e56a5eb3e69/bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableBufferedMutatorHelper.java#L192-L203

Reply:
Looking at this closer, why do we even need the if block or the try block? They are basically no-ops

Reply:
We would need these Result for AbstractBigtableTable#get(List) operation which is HBase's Table method. Internally this batch is used for AbstractBigtbaleTable's batch(List), delete(List), get(List), put(List) operation.

Reply:
I see, please update it to make it clearer:

```java
final Result result;

if (message instanceof Result) {
   // Handle the completion of Gets
   result = (Result) message;
} else {
  // Handle the completion of other operations like Deletes & Puts that don't return anything
   result = Rest.EMPTY_RESULT
}
```

Reply:
The try isn't necessary because nothing can fail